recipes/hidefix/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "hidefix" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hidefix-{{ version }}.tar.gz
+  sha256: c4e8e7196b556a5e2e8da5fedb20f183deb8f4add6fd1f54b303d076577f53e5
+
+build:
+  number: 0
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - cmake
+    - make
+    - pkg-config
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - maturin                             # [build_platform != target_platform]
+    - {{ compiler('c') }}                 # [linux]
+    - {{ compiler('m2w64_c') }}           # [win]
+    - {{ compiler('cxx') }}               # [linux]
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - pip
+    - maturin >=0.15,<0.16
+    - pkg-config
+  run:
+    - python
+    - xz
+    - numpy
+    - xarray
+    - netCDF4
```

Not sure if this will fix Windows but it is worth a try:

```suggestion
    - netCDF4
    - ucrt  # [win]
```

---

I think it is missing the windows cxx runtime.